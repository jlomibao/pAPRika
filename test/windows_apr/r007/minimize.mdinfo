

   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   5000      -8.1131E+02     4.6892E-01     1.5986E+00     H14        51

 BOND    =        1.1113  ANGLE   =      107.7372  DIHED      =       51.4531
 VDWAALS =      -46.2362  EEL     =     1337.5549  EGB        =     -135.9308
 1-4 VDW =        5.2059  1-4 EEL =    -2132.2018  RESTRAINT  =        0.0011
 EAMBER  =     -811.3064
 NMR restraints: Bond =    0.001   Angle =     0.000   Torsion =     0.000
===============================================================================
