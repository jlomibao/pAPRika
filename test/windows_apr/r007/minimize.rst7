defa
  125
  -0.5217299   4.9759794   1.8352743   1.0157616   4.9108507   1.8351994
  -0.8855230   4.2791160   0.6084489  -0.8854172   4.2789157   3.0620207
  -0.9016915   6.0041161   1.8353677   1.4813175   5.9031681   1.8352544
   1.3195413   4.1855829   0.6086108   1.3196403   4.1853805   3.0616443
   2.6868499   4.0833360   0.1137431   2.6869790   4.0831292   3.5564096
   0.2013730   3.8584505   3.7830182   0.2012082   3.8588125  -0.1127335
   0.1802963   3.3487369   4.8903536   0.1800364   3.3493644  -1.2201726
   2.6633019   4.0469851  -0.9815036   3.2459728   4.9656337   0.4459605
   2.6634996   4.0466460   4.6516534   3.2460617   4.9654782   3.2242609
  -4.6359995   1.9609929   1.8353915  -3.8101789   3.2595702   1.8353849
  -4.2146613   1.2971976   0.6086289  -4.2146444   1.2971708   3.0621399
  -5.7162815   2.1465373   1.8353990  -4.4363501   4.1592066   1.8354156
  -3.0303803   3.1595419   0.6085808  -3.0303422   3.1594931   3.0621723
  -2.2566282   4.2914791   0.1136653  -2.2564481   4.2913668   3.5570042
  -3.3067569   2.0275244   3.7832535  -3.3068198   2.0275879  -0.1125037
  -2.8757687   1.7539045   4.8904310  -2.8759291   1.7540344  -1.2197176
  -2.2364135   4.2526310  -0.9815623  -2.7400198   5.2175542   0.4455590
  -2.2360775   4.2525093   4.6522286  -2.7398245   5.2174773   3.2251871
  -4.0834557  -3.1098994   1.8352299  -4.7950372  -1.7453603   1.8352893
  -3.2979908  -3.0769683   0.6084196  -3.2978096  -3.0769768   3.0619283
  -4.7843471  -3.9526255   1.8352733  -5.8872404  -1.8377079   1.8353327
  -4.3184596  -1.1200681   0.6084904  -4.3183348  -1.1201025   3.0620600
  -4.9114482   0.1162920   0.1138291  -4.9113625   0.1161867   3.5568603
  -3.4762384  -1.9255441   3.7830720  -3.4765003  -1.9255179  -0.1126685
  -3.0236717  -1.6891544   4.8902126  -3.0241256  -1.6891314  -1.2198727
  -4.8678312   0.1145033  -0.9814041  -5.9550818   0.1610906   0.4458801
  -4.8677290   0.1142922   4.6520926  -5.9550061   0.1609698   3.2248384
   0.5850516  -5.1654772   1.8348867  -0.9525010  -5.1002335   1.8349968
   0.9488673  -4.4684753   0.6081180   0.9490201  -4.4685769   3.0616662
   0.9649515  -6.1936367   1.8348114  -1.4181211  -6.0925207   1.8349953
  -1.2561428  -4.3749348   0.6082604  -1.2559815  -4.3749922   3.0618120
  -2.6234541  -4.2707873   0.1137634  -2.6232325  -4.2708197   3.5564769
  -0.1375598  -4.0479618   3.7828479  -0.1378169  -4.0478594  -0.1129050
  -0.1162271  -3.5378991   4.8900233  -0.1166372  -3.5378001  -1.2200704
  -2.6002633  -4.2341476  -0.9814814  -3.1837760  -5.1522782   0.4461024
  -2.5999056  -4.2341971   4.6517193  -3.1836166  -5.1522932   3.2241969
   4.6985642  -2.1491158   1.8347278   3.8734279  -3.4481302   1.8347324
   4.2768108  -1.4854514   0.6080365   4.2769043  -1.4855162   3.0614841
   5.7789409  -2.3341082   1.8346785   4.5000428  -4.3474575   1.8346868
   3.0934961  -3.3484263   0.6079936   3.0935898  -3.3484772   3.0615360
   2.3199612  -4.4806099   0.1132956   2.3201839  -4.4807255   3.5562729
   3.3694519  -2.2163778   3.7826302   3.3693443  -2.2163156  -0.1130897
   2.9384312  -1.9430251   4.8898601   2.9383206  -1.9429680  -1.2203045
   2.2996716  -4.4419046  -0.9819356   2.8035488  -5.4065449   0.4452948
   2.3000743  -4.4421230   4.6515111   2.8037428  -5.4066136   3.2241016
   4.1487961   2.9227297   1.8349519   4.8584179   1.5570306   1.8348482
   3.3640816   2.8906172   0.6075482   3.3641844   2.8904798   3.0624216
   4.8510137   3.7643509   1.8349637   5.9507551   1.6477796   1.8347929
   4.3808256   0.9320313   0.6082359   4.3809451   0.9319223   3.0614542
   4.9732725  -0.3044196   0.1130564   4.9734581  -0.3045515   3.5564691
   3.5393915   1.7378983   3.7824786   3.5392557   1.7381168  -0.1126536
   3.0853141   1.5010961   4.8889128   3.0851699   1.5014730  -1.2191023
   4.9290506  -0.3024396  -0.9821521   6.0171134  -0.3493634   0.4444356
   4.9293904  -0.3026534   4.6516840   6.0172496  -0.3494808   3.2249325
   0.0002861   0.0065719  17.9516843  -0.9137705  -0.5722291  17.7708984
   0.8630697  -0.6461868  17.7708420   0.0343550   0.8256765  17.2229123
   0.0234301   0.5616135  19.3893257   0.9307793   1.1618708  19.5293204
  -0.0232208  -0.5613935  20.4476648  -0.8309425   1.2351254  19.5293931
  -0.9305255  -1.1616876  20.3075397   0.8311835  -1.2348560  20.3075535
  -0.0001883  -0.0064939  21.8853896  -0.8630207   0.6462109  22.0661939
  -0.0342624  -0.8256868  22.6140540   0.9138338   0.5723473  22.0662396
  -0.0000784   0.0001617  -6.0007076  -0.0001775  -0.0000563  -9.0000704
   0.0001745   2.2000035 -11.2000237
