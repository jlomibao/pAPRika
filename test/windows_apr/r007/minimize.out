
          -------------------------------------------------------
          Amber 18 SANDER                              2018
          -------------------------------------------------------

| Run on 04/10/2019 at 18:58:18

|   Executable path: sander
| Working directory: /home/jlomibao/pAPRika_fork/pAPRika/test/windows_apr/r007
|          Hostname: entropy.ucsd.edu

  [-O]verwriting output

File Assignments:
|  MDIN: minimize.in                                                           
| MDOUT: minimize.out                                                          
|INPCRD: cb6-but-dum.rst7                                                      
|  PARM: cb6-but-dum.prmtop                                                    
|RESTRT: minimize.rst7                                                         
|  REFC: cb6-but-dum.rst7                                                      
| MDVEL: mdvel                                                                 
| MDFRC: mdfrc                                                                 
|  MDEN: minimize.mden                                                         
| MDCRD: minimize.nc                                                           
|MDINFO: minimize.mdinfo                                                       
|  MTMD: mtmd                                                                  
|INPDIP: inpdip                                                                
|RSTDIP: rstdip                                                                
|INPTRA: inptraj                                                               


 Here is the input file:

PBC MD Simulation                                                              
 &cntrl                                                                        
  imin =          1,                                                           
  ntx =           1,                                                           
  irest =         0,                                                           
  maxcyc =        5000,                                                        
  ncyc =          5000,                                                        
  dt =            0.002,                                                       
  nstlim =        5000,                                                        
  ntpr =          500,                                                         
  ntwe =          500,                                                         
  ntwr =          5000,                                                        
  ntwx =          500,                                                         
  ntxo =          1,                                                           
  ioutfm =        1,                                                           
  ntf =           2,                                                           
  ntc =           2,                                                           
  cut =           999.0,                                                       
  igb =           1,                                                           
  tempi =         298.15,                                                      
  temp0 =         298.15,                                                      
  ntt =           3,                                                           
  gamma_ln =      1.0,                                                         
  ig =            -1,                                                          
  ntp =           0,                                                           
  barostat =      2,                                                           
  ntr =           1,                                                           
  restraint_wt =  50.0,                                                        
  restraintmask = '@DUM',                                                      
  nmropt =        1,                                                           
  pencut =        -1,                                                          
  ntb =           0,                                                           
 /                                                                             
 &wt type = 'END', /                                                           
DISANG = rest.in                                                               
LISTOUT = POUT                                                                 
                                                                               

--------------------------------------------------------------------------------
   1.  RESOURCE   USE: 
--------------------------------------------------------------------------------

| Flags:                                                                        
| New format PARM file being parsed.
| Version =    1.000 Date = 01/16/19 Time = 17:42:13
 NATOM  =     125 NTYPES =       7 NBONH =      46 MBONA  =      93
 NTHETH =     118 MTHETA =     158 NPHIH =     200 MPHIA  =     375
 NHPARM =       0 NPARM  =       0 NNB   =     836 NRES   =       5
 NBONA  =      93 NTHETA =     158 NPHIA =     375 NUMBND =       6
 NUMANG =      12 NPTRA  =      13 NATYP =       7 NPHB   =       0
 IFBOX  =       0 NMXRS  =     108 IFCAP =       0 NEXTRA =       0
 NCOPY  =       0

 Implicit solvent radii are modified Bondi radii (mbondi)                                                   

|     Memory Use     Allocated
|     Real               14705
|     Hollerith            382
|     Integer            27997
|     Max Pairs              1
|     nblistReal             0
|     nblist Int             0
|       Total              225 kbytes

| Note: 1-4 EEL scale factors are being read from the topology file.

| Note: 1-4 VDW scale factors are being read from the topology file.
| Duplicated    0 dihedrals
| Duplicated    0 dihedrals
Note: ig = -1. Setting random seed to   620863 based on wallclock time in microseconds.

--------------------------------------------------------------------------------
   2.  CONTROL  DATA  FOR  THE  RUN
--------------------------------------------------------------------------------

default_name                                                                    

General flags:
     imin    =       1, nmropt  =       1

Nature and format of input:
     ntx     =       1, irest   =       0, ntrx    =       1

Nature and format of output:
     ntxo    =       1, ntpr    =     500, ntrx    =       1, ntwr    =    5000
     iwrap   =       0, ntwx    =     500, ntwv    =       0, ntwe    =     500
     ioutfm  =       1, ntwprt  =       0, idecomp =       0, rbornstat=      0

Potential function:
     ntf     =       2, ntb     =       0, igb     =       1, nsnb    =      25
     ipol    =       0, gbsa    =       0, iesp    =       0
     dielc   =   1.00000, cut     = 999.00000, intdiel =   1.00000
     saltcon =   0.00000, offset  =   0.09000, gbalpha=    1.00000
     gbbeta  =   0.00000, gbgamma =   0.00000, surften =   0.00500
     rdt     =   0.00000, rgbmax  =  25.00000  extdiel =  78.50000
     alpb  =        0

Frozen or restrained atoms:
     ibelly  =       0, ntr     =       1
     restraint_wt =  50.00000

Energy minimization:
     maxcyc  =    5000, ncyc    =    5000, ntmin   =       1
     dx0     =   0.01000, drms    =   0.00010

SHAKE:
     ntc     =       2, jfastw  =       0
     tol     =   0.00001

NMR refinement options:
     iscale  =       0, noeskp  =       1, ipnlty  =       1, mxsub   =       1
     scalm   = 100.00000, pencut  =  -1.00000, tausw   =   0.10000
|  INFO: Old style inpcrd file read


    LOADING THE CONSTRAINED ATOMS AS GROUPS


   5.  REFERENCE ATOM COORDINATES

      
     Mask @DUM; matches     3 atoms

--------------------------------------------------------------------------------
   3.  ATOMIC COORDINATES AND VELOCITIES
--------------------------------------------------------------------------------

                                                                                
 begin time read from input coords =     0.000 ps



           Begin reading energy term weight changes/NMR restraints
 WEIGHT CHANGES:
                         ** No weight changes given **

 RESTRAINTS:
 Requested file redirections:
  DISANG    = rest.in
  LISTOUT   = POUT
 Restraints will be read from file: rest.in
Here are comments from the DISANG input file:

                       Number of restraints read =     9

                  Done reading weight changes/NMR restraints


 Number of triangulated 3-point waters found:        0

--------------------------------------------------------------------------------
   4.  RESULTS
--------------------------------------------------------------------------------



   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
      1      -7.9698E+02     1.0074E+01     2.4798E+01     N3          8

 BOND    =       11.0328  ANGLE   =      109.3438  DIHED      =       52.9822
 VDWAALS =      -45.6645  EEL     =     1326.5993  EGB        =     -129.6177
 1-4 VDW =        5.8710  1-4 EEL =    -2127.5276  RESTRAINT  =        0.0000
 EAMBER  =     -796.9808
 NMR restraints: Bond =    0.000   Angle =     0.000   Torsion =     0.000
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    500      -8.1131E+02     4.6901E-01     1.5985E+00     H14        51

 BOND    =        1.1095  ANGLE   =      107.7363  DIHED      =       51.4553
 VDWAALS =      -46.2317  EEL     =     1337.5031  EGB        =     -135.9178
 1-4 VDW =        5.2035  1-4 EEL =    -2132.1646  RESTRAINT  =        0.0011
 EAMBER  =     -811.3064
 NMR restraints: Bond =    0.001   Angle =     0.000   Torsion =     0.000
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   1000      -8.1131E+02     4.6900E-01     1.5986E+00     H14        51

 BOND    =        1.1098  ANGLE   =      107.7365  DIHED      =       51.4550
 VDWAALS =      -46.2322  EEL     =     1337.5093  EGB        =     -135.9194
 1-4 VDW =        5.2038  1-4 EEL =    -2132.1692  RESTRAINT  =        0.0011
 EAMBER  =     -811.3065
 NMR restraints: Bond =    0.001   Angle =     0.000   Torsion =     0.000
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   1500      -8.1131E+02     4.6899E-01     1.5986E+00     H14        51

 BOND    =        1.1100  ANGLE   =      107.7365  DIHED      =       51.4548
 VDWAALS =      -46.2327  EEL     =     1337.5150  EGB        =     -135.9208
 1-4 VDW =        5.2040  1-4 EEL =    -2132.1733  RESTRAINT  =        0.0011
 EAMBER  =     -811.3065
 NMR restraints: Bond =    0.001   Angle =     0.000   Torsion =     0.000
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   2000      -8.1131E+02     4.6898E-01     1.5986E+00     H14        51

 BOND    =        1.1102  ANGLE   =      107.7366  DIHED      =       51.4545
 VDWAALS =      -46.2332  EEL     =     1337.5207  EGB        =     -135.9222
 1-4 VDW =        5.2043  1-4 EEL =    -2132.1773  RESTRAINT  =        0.0011
 EAMBER  =     -811.3065
 NMR restraints: Bond =    0.001   Angle =     0.000   Torsion =     0.000
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   2500      -8.1131E+02     4.6897E-01     1.5986E+00     H14        51

 BOND    =        1.1104  ANGLE   =      107.7367  DIHED      =       51.4543
 VDWAALS =      -46.2337  EEL     =     1337.5266  EGB        =     -135.9237
 1-4 VDW =        5.2045  1-4 EEL =    -2132.1817  RESTRAINT  =        0.0011
 EAMBER  =     -811.3065
 NMR restraints: Bond =    0.001   Angle =     0.000   Torsion =     0.000
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   3000      -8.1131E+02     4.6896E-01     1.5986E+00     H14        51

 BOND    =        1.1105  ANGLE   =      107.7368  DIHED      =       51.4540
 VDWAALS =      -46.2342  EEL     =     1337.5322  EGB        =     -135.9251
 1-4 VDW =        5.2048  1-4 EEL =    -2132.1855  RESTRAINT  =        0.0011
 EAMBER  =     -811.3064
 NMR restraints: Bond =    0.001   Angle =     0.000   Torsion =     0.000
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   3500      -8.1131E+02     4.6895E-01     1.5986E+00     H14        51

 BOND    =        1.1107  ANGLE   =      107.7369  DIHED      =       51.4538
 VDWAALS =      -46.2347  EEL     =     1337.5378  EGB        =     -135.9266
 1-4 VDW =        5.2051  1-4 EEL =    -2132.1894  RESTRAINT  =        0.0011
 EAMBER  =     -811.3063
 NMR restraints: Bond =    0.001   Angle =     0.000   Torsion =     0.000
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   4000      -8.1131E+02     4.6894E-01     1.5986E+00     H14        51

 BOND    =        1.1109  ANGLE   =      107.7371  DIHED      =       51.4535
 VDWAALS =      -46.2352  EEL     =     1337.5438  EGB        =     -135.9281
 1-4 VDW =        5.2053  1-4 EEL =    -2132.1939  RESTRAINT  =        0.0011
 EAMBER  =     -811.3065
 NMR restraints: Bond =    0.001   Angle =     0.000   Torsion =     0.000
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   4500      -8.1131E+02     4.6893E-01     1.5986E+00     H14        51

 BOND    =        1.1111  ANGLE   =      107.7371  DIHED      =       51.4533
 VDWAALS =      -46.2357  EEL     =     1337.5494  EGB        =     -135.9294
 1-4 VDW =        5.2056  1-4 EEL =    -2132.1979  RESTRAINT  =        0.0011
 EAMBER  =     -811.3065
 NMR restraints: Bond =    0.001   Angle =     0.000   Torsion =     0.000
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   5000      -8.1131E+02     4.6892E-01     1.5986E+00     H14        51

 BOND    =        1.1113  ANGLE   =      107.7372  DIHED      =       51.4531
 VDWAALS =      -46.2362  EEL     =     1337.5549  EGB        =     -135.9308
 1-4 VDW =        5.2059  1-4 EEL =    -2132.2018  RESTRAINT  =        0.0011
 EAMBER  =     -811.3064
 NMR restraints: Bond =    0.001   Angle =     0.000   Torsion =     0.000
===============================================================================


  Maximum number of minimization cycles reached.


                    FINAL RESULTS



   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   5000      -8.1131E+02     4.6892E-01     1.5986E+00     H14        51

 BOND    =        1.1113  ANGLE   =      107.7372  DIHED      =       51.4531
 VDWAALS =      -46.2362  EEL     =     1337.5549  EGB        =     -135.9308
 1-4 VDW =        5.2059  1-4 EEL =    -2132.2018  RESTRAINT  =        0.0011
 EAMBER  =     -811.3064
 NMR restraints: Bond =    0.001   Angle =     0.000   Torsion =     0.000
===============================================================================
 ------------------------------------------------------------------------------


 Final Restraint Analysis for coords: minimize.rst7                           


 Restraints, deviations, and energy contributions:    pencut =   -1.00

 ------------------------------------------------------------------------------
     First atom        Last atom    curr. value target deviation  penalty
 ------------------------------------------------------------------------------
  DUM  DM1    3 --  C7   CB6    1:    9.303    9.302    0.001    0.000 d    0: 0
  DUM  DM2    4 --  C7   CB6    1:  147.388  147.398    0.010    0.000 a
  DUM  DM2    4 --  DUM  DM1    3:   49.472   49.483    0.011    0.000 t
  DUM  DM1    3 --  C31  CB6    1:   62.001   62.003    0.002    0.000 a
  DUM  DM1    3 --  C7   CB6    1: -146.190 -146.178    0.011    0.000 t
  C7   CB6    1 --  C31  CB6    1:  -72.543  -72.548    0.005    0.000 t
  DUM  DM1    3 --  C    BUT    2:   23.952   24.000    0.048    0.001 d    0: 0
  DUM  DM2    4 --  C    BUT    2:  179.989  180.000    0.011    0.000 a
  DUM  DM1    3 --  C3   BUT    2:  179.794  180.000    0.206    0.000 a
                                       Total distance penalty:      0.001
                                       Total angle    penalty:      0.000
                                       Total torsion  penalty:      0.000
|                               RMS deviation from ideal bonds :      0.0094
|                               RMS deviation from ideal angles:      4.472
 ------------------------------------------------------------------------------

--------------------------------------------------------------------------------
   5.  TIMINGS
--------------------------------------------------------------------------------

|                Calc gb radii              2.13 (21.47% of Gen B)
|                Calc gb diag               3.86 (38.87% of Gen B)
|                Calc gb off-diag           3.93 (39.63% of Gen B)
|                Other                      0.00 ( 0.02% of Gen B)
|             Gen Born time              9.93 (100.0% of Nonbo)
|          Nonbond force              9.93 (90.30% of Force)
|          Bond/Angle/Dihedral        1.03 ( 9.39% of Force)
|          Other                      0.03 ( 0.31% of Force)
|       Force time                10.99 (100.0% of Runmd)
|    Runmd Time                10.99 (99.81% of Total)
|    Other                      0.02 ( 0.19% of Total)
| Total time                11.01 (100.0% of ALL  )

| Number of list builds   :          0

| Highest rstack allocated:          0
| Highest istack allocated:          0
|           Job began  at 18:58:18.617  on 04/10/2019
|           Setup done at 18:58:18.622  on 04/10/2019
|           Run   done at 18:58:29.629  on 04/10/2019
|     wallclock() was called   90028 times

|3D-RISM memory allocation summary
|Type          Maximum        Current   
|Integer       0.00000 GB     0.00000 GB
|Real          0.00000 GB     0.00000 GB
|Logical       0.00000 GB     0.00000 GB
|Character     0.00000 GB     0.00000 GB
|---------------------------------------
|Total         0.00000 GB     0.00000 GB
